package.json

```diff
@@ -34,7 +34,6 @@
     "packages/sharing-service",
     "packages/zoe",
     "packages/cosmic-swingset",
-    "packages/generator-agoric-dapp",
```

Comment:
the rest seems ok to me, but why is this removal in this PR?

Reply:
Agh.  Leakage.

I'll revert it.